.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: csharp
+mono: none
+
+notifications:
+  email: false
+
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+    - os: linux 
+      dist: trusty # Ubuntu 14.04
+      dotnet: 2.1.2
+    - os: osx
+      dotnet: 2.1.2
```

Comment:
No need to repeat the `dotnet: 2.1.2` here for both dist. Just put it under the beginning, like here: https://github.com/nopara73/MagicalCryptoWallet/blob/master/.travis.yml

Reply:
Sure, however I prefer to keep them explicitly specified just in case we need have different dotnet sdk for linux and osx.